### ISSUE TITLE
Unable to import `bynder_sdk.BynderClient` in version 1.1.3

### ISSUE THREAD

**Issue by rahulsabbineni** [Bynder/bynder-python-sdk#15]
After installing latest via `pip install bynder-sdk`:

```python
>>> from bynder_sdk import BynderClient
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/dev/Desktop/internal-tools/dashboard/venv/lib/python3.7/site-packages/bynder_sdk/__init__.py", line 1, in <module>
    from bynder_sdk.client.bynder_client import BynderClient # noqa
  File "/Users/dev/Desktop/internal-tools/dashboard/venv/lib/python3.7/site-packages/bynder_sdk/client/bynder_client.py", line 6, in <module>
    from bynder_sdk.oauth2 import BynderOAuth2Session
  File "/Users/dev/Desktop/internal-tools/dashboard/venv/lib/python3.7/site-packages/bynder_sdk/oauth2.py", line 6, in <module>
    from bynder_sdk.util import SessionMixin
  File "/Users/dev/Desktop/internal-tools/dashboard/venv/lib/python3.7/site-packages/bynder_sdk/util.py", line 1, in <module>
    with open('VERSION') as fh:
FileNotFoundError: [Errno 2] No such file or directory: 'VERSION'
```

To resolve this issue, I downgraded to version 1.1.2 and I was able to import `BynderClient` and successfully make API calls.

**Comment by jesstucker** [comment]
Also having this err

**Comment by WouterToering** [comment]
Hi @sighduck1, @jesstucker and @mackenzieclark,

Apologies for releasing the broken build and for the late reaction on this issue. We've just released version 1.1.4 which resolves this issue.
